include/dpp/queues.h

```diff
@@ -70,32 +108,65 @@ enum http_error {
  * rate limit figures, and returned request body.
  */
 struct DPP_EXPORT http_request_completion_t {
-	/** @brief HTTP headers of response */
+	/**
+	 * @brief HTTP headers of response.
+	 */
 	std::multimap<std::string, std::string> headers;
-	/** @brief HTTP status, e.g. 200 = OK, 404 = Not found, 429 = Rate limited */
+
+	/**
+	 * @brief HTTP status (e.g. 200 = OK, 404 = Not found, 429 = Rate limited, etc).
```

Comment:
where we have a longer description like this its better to split it onto multiple lines. The second part of the comment will be split below the title in doxygen. e.g.
```cpp
/**
 * @brief HTTP status
 * e.g. 200 = OK, 404 = Not found, 429 = Rate limited, etc.
 */
```